composer.json

```diff
@@ -25,6 +25,9 @@
         "ocramius/proxy-manager": "~0.5"
     },
     "require-dev": {
+        "ext-mongo" : "*",
```

Comment:
I've checked the ``doctrine/mongodb-odm`` and [didn't found it](https://github.com/doctrine/mongodb-odm/blob/master/composer.json) that's why I've put it there explicitly.

But looks like it's required by ````doctrine/mongodb`` after all, which is a requirement of ``doctrine/mongodb-odm``. I'll be removing this line...